source/index.rst

```diff
@@ -3,13 +3,25 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Tools for credible digital twins's documentation!
-============================================================
+Tools for credible modelling of digital twins
+=============================================
+
+This is a Sphinx placeholder index page for the EMBC'23 workshop docs.
+It will not be displayed on the resulting website at: https://tools-for-credible-digital-twins.github.io/.
```

Comment:
Use this index page for rendering and navigating the content in your local working environment.

Reply:
see 7ba4224